### ISSUE TITLE
support pytest-bdd>=6.1

### ISSUE THREAD

**Issue by mabdinur** [DataDog/dd-trace-py#4492]
### Which version of dd-trace-py are you using?

Effected versions: `ddtrace>=1.3`

### Which version of pip are you using?

Not applicable

### Which libraries and their versions are you using?

``pytest-bdd>=6.1``

### How can we reproduce your problem?

Run ddtrace pytest-bdd tests with  ``pytest-bdd>=6.1``. 

First Failure
- Sample Error: https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/23378/workflows/7d24fef6-27cf-4bc9-959a-b8df7cbd64c2/jobs/1590909
- Cause: pytest_bdd.__version__ was removed.

Second Failure
- Sample Error: https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/23379/workflows/b7bbdb4e-e0df-4f68-8c2b-2b6a320d9c9f/jobs/1591008 
- Cause:  ddtrace uses accesses [step_func._pytest_bdd_parsers](https://github.com/DataDog/dd-trace-py/blob/v1.6.0/ddtrace/contrib/pytest_bdd/plugin.py#L80) to parse pytest arguments. This attribute was [removed in v6.1](https://github.com/pytest-dev/pytest-bdd/pull/534/files)

### What is the result that you get?

CI failures

### What is the result that you expected?

No CI Failures

**Comment by nsfrias** [comment]
Hi all,

Is there a workaround for this issue? 

I am still getting the error:

(stack excerpt)
```
AttributeError: module 'pytest_bdd' has no attribute '__version__'
```

**Comment by nsfrias** [comment]
@majorgreys Thank you for addressing this issue.

Do you have any idea when the build is going to be released ?

**Comment by gnufede** [comment]
> @majorgreys Thank you for addressing this issue.
> 
> Do you have any idea when the build is going to be released ?

You can test already it by using this RC: https://pypi.org/project/ddtrace/1.17.0rc1/

**Comment by romainkomorndatadog** [comment]
@nsfrias Is the repo you're using our `pytest-bdd` plugin with public, by any chance? 

I created a small test repo for myself to test with this release, but it only has a couple of simple scenarios. I've also tested with some bits of https://github.com/AutomationPanda/behavior-driven-python , but that repo hasn't seen activity in a couple of years.

Having something to test against that's both substantial and recent would be great.
